src/Shapes.tsx

```diff
@@ -0,0 +1,121 @@
+import React, { MouseEvent } from "react";
+
+function circle(
+    x: number,
+    y: number,
+    setDragItem?: (e: Draggable) => void
+): React.JSX.Element {
```

Comment:
Probably we want to convert this factories into components so we can use hooks inside

Reply:
Done, now the shapes are React components